clientv3/yaml/config.go

```diff
@@ -33,7 +33,8 @@ type yamlConfig struct {
 	InsecureSkipTLSVerify bool   `json:"insecure-skip-tls-verify"`
 	Certfile              string `json:"cert-file"`
 	Keyfile               string `json:"key-file"`
-	CAfile                string `json:"ca-file"`
+	TrustedCAfile         string `json:"trusted-ca-file"`
+	CAfile                string `json:"ca-file"` // TODO: deprecate this in v4
```

Comment:
probably add a comment on top of the field. saying that this field is deprecated, use TrustedCAfile field instead.